### ISSUE TITLE
How to include and / or setting the remote ip

### ISSUE THREAD

**Issue by WebNetSolving** [hCaptcha/hcaptcha-wordpress-plugin#64]
Hi,

How to include and / or setting the remote ip ?

Thanks in advance for the support.

**Comment by e271828-** [comment]
You'd just add it here:
https://github.com/hCaptcha/hcaptcha-wordpress-plugin/blob/19553ba264605fee78b6807b8245e34be90e6e75/common/request.php#L23

i.e. add `&remoteip=$ip` to api.js

with a function like:

```
function get_user_ip() {
	// NOTE: this must be adjusted to the server environment in order to be secure.
	// For example, if you are using Cloudflare, you should be checking for HTTP_CF_CONNECTING_IP
	// **first**.

	if ( ! empty( $_SERVER['HTTP_CLIENT_IP'] ) ) {
		$ip = $_SERVER['HTTP_CLIENT_IP'];
	} elseif ( ! empty( $_SERVER['HTTP_CF_CONNECTING_IP'] ) ) {
		// cloudflare
		$ip = $_SERVER['HTTP_CF_CONNECTING_IP'];
	} elseif ( ! empty( $_SERVER['HTTP_X_FORWARDED_FOR'] ) ) {
		$ip = $_SERVER['HTTP_X_FORWARDED_FOR'];
	} else {
		$ip = $_SERVER['REMOTE_ADDR'];
	}

	return $ip;
}
```

**Comment by kagg-design** [comment]
Included in V1.13.0, will be published at the end of September.
